vaults-monitor/README.md

```diff
@@ -0,0 +1,47 @@
+# Vaults Monitor Service
+
+This service periodically checks the status of the bitcoin vaults used by the Hemi tunnel. It will send messages to Slack if any problem is found.
+
+## Components
+
+| Component | Description                                                  |
+| --------- | ------------------------------------------------------------ |
+| `api`     | Node HTTP server to retrieve the status of the vaults.       |
+| `cron`    | Node cron job that analyzes the status and emits the alerts. |
+
+### Configuration
+
+These control the `api` server:
+
+| Variable      | Description                                                     | Default                |
+| ------------- | --------------------------------------------------------------- | ---------------------- |
+| CACHE_MINUTES | The time to cache the status in minutes.                        | 5                      |
+| CHAIN         | The Hemi chain to monitor: mainnet or testnet.                  | mainnet                |
+| ORIGINS       | Comma-separated list of allowed origins. '\*' is not supported. | `http://localhos:3000` |
+| PORT          | The HTTP port the server listens for requests.                  | 3003                   |
```

Comment:
It's unlikely that both are running at the same time, but should we default to port 3004? 3003 is used by the [subgraph-api](https://github.com/hemilabs/ui-monorepo/blob/079326424dc51f418d4c34f8a5a924d029e61b49/subgraph-api/config/default.json#L4)

Reply:
Oh! I thought 3003 was free. Changing to 3004...